### PR TITLE
Adds docs for how to track merge tasks with Trello

### PR DESCRIPTION
**Pull request by lundjordan**

### CODE REVIEW

**Comment by lundjordan** [comment]
some of this will change depending on the result of merge reliability improvements and automation work in flight

File: docs/mergeduty/howto.md [hunk 1]
```diff
@@ -52,6 +54,36 @@ it in the instance list, hover over to `Actions` -> `Instance state` -> `Start`.
...
+  a. In the "Postmortem Action Items" list, select "move all cards in this list" to the "Archived Postmortem Action Items" list
+  
+Now prep the board for this cycle's planned merges:
+
+1. for each card in the "Templates" list, select the card then under Actions, choose "Move" and put in the "Merge Tasks" list
+1. for each newly copied card in the "Merge Tasks" list
+  a. add the people on mergeduty to the members list
+  a. set a deadline for each that correspond with the current [merge schedule](https://calendar.google.com/calendar/embed?src=bW96aWxsYS5jb21fZGJxODRhbnI5aTh0Y25taGFiYXRzdHY1Y29AZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+
+For each merge task:
```
Comment (MihaiTabara) [inline docs/mergeduty/howto.md]: Nit: might be worth bolding this heading?

File: docs/mergeduty/howto.md [hunk 2]
```diff
@@ -52,6 +54,36 @@ it in the instance list, hover over to `Actions` -> `Instance state` -> `Start`.
...
+
+As you go through the steps documented below, update the checklists within the cards under "Merge Tasks". This helps with handoff and tracking state.
+
+As issues arise, add a card under the "Merge Issues" list. Associate with bugs where appropriate. Use labels for issues. Each issue should either be "Resolved" or "Unresolved" and some "Resolved" issues may be also labeled as "Future Threats".
+
+For postmortems:
```
Comment (MihaiTabara) [inline docs/mergeduty/howto.md]: Nit: might be worth bolding this heading?

**Comment by MihaiTabara** [comment]
On a separate note, @escapewindow added some good ideas on how to better approach mergeduty tasks in https://bugzilla.mozilla.org/show_bug.cgi?id=1499047#c1. If we end up down that path, we should revisit this documentation and trello card and adjust.

**Comment by lundjordan** [comment]
Thanks for the review!

> On a separate note, @escapewindow added some good ideas on how to better approach mergeduty tasks in https://bugzilla.mozilla.org/show_bug.cgi?id=1499047#c1. If we end up down that path, we should revisit this documentation and trello card and adjust.

Yep, that was what I meant with first comment in this PR

File: docs/mergeduty/howto.md [hunk 3]
```diff
@@ -52,6 +54,36 @@ it in the instance list, hover over to `Actions` -> `Instance state` -> `Start`.
...
 
+### Set up mergeduty Trello board
+
+Rather than extend Releasewarrior with more complexity, the idea here is to try Trello and use Templated cards for todo tracking.
+
+To track human tasks and issues during merges, we use the following [trello board](https://trello.com/b/AyyFAEbS/mergeduty-tasks)
+
+First, ensure the board is clean:
+
+1. In the "Merge Tasks" list, select "Archive all cards in this list"
```
[the remark this answers is not in the document]
Comment (lundjordan) [reply docs/mergeduty/howto.md]: will do

File: docs/mergeduty/howto.md [hunk 4]
```diff
@@ -52,6 +54,36 @@ it in the instance list, hover over to `Actions` -> `Instance state` -> `Start`.
...
+1. sanity check that all items in "Postmortem Issues" and "Postmortem Action Items" lists have been resolved then:
+  a. In the "Postmortem Issues" list, select "move all cards in this list" to the "Archived Postmortem Issues" list
```
[the remark this answers is not in the document]
Comment (lundjordan) [reply docs/mergeduty/howto.md]: fixing

File: docs/mergeduty/howto.md [hunk 5]
```diff
@@ -52,6 +54,36 @@ it in the instance list, hover over to `Actions` -> `Instance state` -> `Start`.
...
+  a. In the "Postmortem Action Items" list, select "move all cards in this list" to the "Archived Postmortem Action Items" list
+  
+Now prep the board for this cycle's planned merges:
+
+1. for each card in the "Templates" list, select the card then under Actions, choose "Move" and put in the "Merge Tasks" list
```
[the remark this answers is not in the document]
Comment (lundjordan) [reply docs/mergeduty/howto.md]: yep! I meant copy. Thanks for catching.

File: docs/mergeduty/howto.md (hunk 2 above)
Comment (lundjordan) [reply docs/mergeduty/howto.md]: will do

**Comment by MihaiTabara** [comment]
Ah, I missed the first comment, makes sense :)

**Pull request merged**